group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupConfig.java

```diff
@@ -63,6 +71,18 @@ public class GroupConfig extends AbstractConfig {
             atLeast(1),
             MEDIUM,
             GroupCoordinatorConfig.CONSUMER_GROUP_HEARTBEAT_INTERVAL_MS_DOC)
+        .define(SHARE_SESSION_TIMEOUT_MS_CONFIG,
+            INT,
+            GroupCoordinatorConfig.SHARE_GROUP_SESSION_TIMEOUT_MS_DEFAULT,
+            atLeast(1),
```

Comment:
KIP defines `share.min.session.timeout.ms ` as 45 secs but we do not have any range defined for `share.session.timeout.ms`. Though I understand that in other places as well (`heartbeat.interval.ms`) there is no valid range defined but just writing it down to get the clarification.